src/popup/components/AdvancedTransactionDetails.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <DetailsItem
+    expandable
+    class="advanced-transaction-details"
+    :label="$t('transaction.advancedDetails')"
+  >
+    <DetailsItem
+      v-if="callData"
+      :label="$t('transaction.data')"
+    >
+      <Tabs class="tabs">
+        <Tab
+          v-for="({ name, text }) in dataTabs"
+          :key="name"
+          :data-cy="name"
+          :name="name"
+          :text="text"
+          :active="activeTab === name"
+          @click="setActiveTab(name)"
+        />
+      </Tabs>
+      <div class="tabs-content">
+        <!-- Decoded Data -->
+        <DetailsItem v-if="activeTab === dataTabs[0].name">
+          <Panel v-if="decodedCallData?.args">
+            <PanelTableItem
+              v-for="([key, value]) in Object
+                .entries(decodedCallData?.args as any) || []"
+              :key="key"
+              :name="key"
+            >
+              <div class="scrollable">
+                {{ value }}
+              </div>
+            </PanelTableItem>
+          </Panel>
+          <InfoBox
+            v-else
+            type="warning"
+            :text="$t('transaction.decodingDataFailed')"
+          />
+        </DetailsItem>
+        <!-- Raw Data -->
+        <DetailsItem
+          v-if="activeTab === dataTabs[1].name"
+          :value="callData"
+        />
+      </div>
+    </DetailsItem>
+
+    <DetailsItem
+      v-if="transactionArguments"
+      :label="$t('modals.confirmTransactionSign.arguments')"
+      :value="transactionArguments"
+    />
+    <slot name="extra" />
+  </DetailsItem>
+</template>
+
+<script lang="ts">
+import {
+  computed,
+  defineComponent,
+  PropType,
+  ref,
+} from 'vue';
+import { useI18n } from 'vue-i18n';
+
+import JsonBig from '@/lib/json-big';
+
+import Panel from '@/popup/components/Panel.vue';
+import PanelTableItem from '@/popup/components/PanelTableItem.vue';
+import Tabs from '@/popup/components/tabs/Tabs.vue';
+import Tab from '@/popup/components/tabs/Tab.vue';
+import InfoBox from '@/popup/components/InfoBox.vue';
+import DetailsItem from '@/popup/components/DetailsItem.vue';
+
+export default defineComponent({
+  name: 'AdvancedTransactionDetails',
+  components: {
+    Panel,
+    PanelTableItem,
+    Tabs,
+    Tab,
+    InfoBox,
+    DetailsItem,
+  },
+  props: {
+    callData: { type: String, default: null },
+    decodedCallData: { type: Object as PropType<any>, default: null },
```

Comment:
I think the `PropType<any>` is a mistake as the type is initially set as an `Object`. Maybe instead of `any` you wanted to use something like `Dictionary`?

Also what do you think about changing the prop name to `callDataDecoded` ?